keyboards/k_type/keymaps/default/config.h

```diff
@@ -0,0 +1,80 @@
+#pragma once
```

Comment:
This config should probably go into the keyboard's config.h file, rather than here.

Otherwise, this file requires a license header.